exchanges/graphcache/src/store/store.ts

```diff
@@ -209,4 +209,22 @@ export class Store implements Cache {
   ): void {
     writeFragment(this, formatDocument(fragment), data, variables as any);
   }
+
+  link(
+    entity: Data | string,
+    field: string,
+    entities: Data | Data[],
+    fieldArgs?: Variables
+  ): void {
+    const entityKey = this.keyOfEntity(entity);
+    if (!entityKey) return;
+
+    InMemoryData.writeLink(
```

Comment:
We can probably create a variant of `writeField` here; my main concern being that embedded entities should work and that repeatedly nested lists should work too 🤔 https://github.com/FormidableLabs/urql/blob/ef4b5b002d31e17d84fb41e5feb24d2200c9656f/exchanges/graphcache/src/operations/write.ts#L305

We don't want to add `filter(Boolean)` either since `null` in a list is valid, although we may want to check that against the schema. Let's see what the current logic is on that. We may want to write `undefined` for schema awareness of a schema is present.